src/Hint/Extensions.hs

```diff
@@ -515,19 +515,21 @@ addDerives nt _ xs = mempty
 derives :: Located HsModule -> Derives
 derives (L _ m) =  mconcat $ map decl (childrenBi m) ++ map idecl (childrenBi m)
   where
-    idecl :: DataFamInstDecl GhcPs -> Derives
-    idecl (DataFamInstDecl (HsIB _ FamEqn {feqn_rhs=HsDataDefn {dd_ND=dn, dd_derivs=(L _ ds)}})) = g dn ds
+    idecl :: LocatedA (DataFamInstDecl GhcPs) -> Derives
```

Comment:
You should remove the `LocatedA` here. Then the test should pass.

Reply:
outstanding @zliu41 , many thanks. i have restored the (now passing) test!